Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -511,17 +511,20 @@ - (void)layoutIfNeeded
 - (BOOL)isOpaque
 {
   _bridge_prologue_read;
-  return _getFromLayer(opaque);
+  return _getFromViewOrLayer(opaque, opaque);
 }
 
+
 - (void)setOpaque:(BOOL)newOpaque
 {
   _bridge_prologue_write;
-  
   BOOL shouldApply = ASDisplayNodeShouldApplyBridgedWriteToView(self);
   
   if (shouldApply) {
     BOOL oldOpaque = _layer.opaque;
```

Comment:
Can we rely on the layer for the old value? Should we get it from the view if it's view-backed?

Maybe this is ok because in this setter, we always set to the layer. But then clients may actually reach out to the layer and set it directly there. I think the safest approach would be to do similar to the getter (get from the view if it's there).